db/schema.rb

```diff
@@ -49,18 +49,17 @@
     t.index ["coronavirus_page_id"], name: "index_coronavirus_sub_sections_on_coronavirus_page_id"
   end
 
-  create_table "coronavirus_timeline_entries", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8", force: :cascade do |t|
+  create_table "coronavirus_timeline_entries", charset: "utf8", force: :cascade do |t|
     t.bigint "coronavirus_page_id", null: false
     t.text "content", null: false
     t.string "heading", null: false
     t.integer "position", null: false
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.string "national_applicability"
```

Comment:
I don't think you are meaning to lose this - you may have gone back too far?

Reply:
😆 how is this happening!

Reply:
Beat me to it!

Reply:
Okay, now looking healthy 😬